Package.resolved

```diff
@@ -6,17 +6,35 @@
         "repositoryURL": "https://github.com/kylebrowning/APNSwift.git",
         "state": {
           "branch": null,
-          "revision": "03e83e2332d13a3c06d5cd70948166e5724c43c7",
-          "version": "3.0.0"
+          "revision": "b53fd38f5611c6b935427febb43d91b15be26c0d",
+          "version": "3.2.0"
         }
       },
       {
         "package": "Apodini",
         "repositoryURL": "https://github.com/Apodini/Apodini.git",
         "state": {
           "branch": null,
-          "revision": "e6a943b0db29b2df5c0c139ab993e57752bf22b7",
-          "version": "0.5.0"
+          "revision": "3738aa97daa1ceac2c8b6979c996e3b86e6eaedc",
+          "version": "0.5.2"
```

Comment:
@hendesi We should update the deployment provider to Apodini 0.6.2 by updating the Package.swift file to work with Apodini 0.6.X.